inspect-f.js

```diff
@@ -31,20 +31,20 @@
     );
   }
 
-  const RSPACE = /^ */;
-  const RCODE = /\s*[^\s]/;
-  const RTABS = /\t/g;
-  const REOL = /\n\r?/;
+  var RSPACE = /^ */;
+  var RCODE = /\s*[^\s]/;
+  var RTABS = /\t/g;
+  var REOL = /\n\r?/;
 
   function getPadding(line){
     return line.match(RSPACE)[0].length;
   }
 
   function guessIndentation(lines){
-    const filtered = lines.filter(x => RCODE.test(x));
-    const paddings = filtered.map(getPadding);
-    const depth = paddings.reduce((a, b) => Math.min(a, b), Infinity);
-    const tabsize = paddings.map(x => x - depth).find(x => x > 1) || 2;
+    var filtered = lines.filter(x => RCODE.test(x));
+    var paddings = filtered.map(getPadding);
+    var depth = paddings.reduce((a, b) => Math.min(a, b), Infinity);
+    var tabsize = paddings.map(x => x - depth).find(x => x > 1) || 2;
```

Comment:
That's odd. The arrow functions in the unit tests were giving me syntax errors in ESLint.